app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+//REQUIRE IN ALL CONTROLLER FUNCS
+const { getTopics } = require("./controllers/news.controllers")
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics)
+
+app.use('/*', (req, res, next) => {
+    res.status(404).send({msg : 'Not Found'});
+})
+
```

Comment:
This could be me being picky, but have the same format for the messages your sending back, one has upper case one doesn't

Reply:
good spot! Thanks v much. Re-factored in the latest commit :)